app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -157,6 +160,15 @@ description: |
   elsif nic_delay
     options.push("nicdelay=#{nic_delay} linksleep=#{nic_delay}")
   end
+
+  # add user specified custom options
+  if custom_options.presence
+    if custom_options.respond_to?('join')
+      options.concat(custom_options)
+    else
+      options += custom_options
```

Comment:
In Ruby you can use `+=` on array to add another array or `<<` to add an element. Here it's probably a string, so you should use `<<` (or `.push`, which is the same).
```suggestion
      options << custom_options
```

Reply:
Thanks, I really don't know the right way to do much in`ruby`!